### PR TITLE
Add Home page

### PR DESCRIPTION
**Pull request by IanCStewart**

### CODE REVIEW

File: src/css/pages/home.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
+body {
+  padding: 0 25px;
+}
+
+ul {
+  margin: 0 auto;
+  padding: 0;
+  list-style: none;
+  width: 85%;
+  max-width: 850px
```
Comment (dandevri) [inline src/css/pages/home.css]: Why u use pixels bruh

**Review by dandevri — DISMISSED** [review]
Please change `px` units to relative units.

File: src/css/pages/home.css (hunk 1 above)
Comment (IanCStewart) [reply src/css/pages/home.css]: for funs

**Review by dandevri — APPROVED** [review]
🌮

**Pull request merged**